contracts/scope/src/msg.rs

```diff
@@ -3,19 +3,19 @@ use provwasm_std::Scope;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
```

Comment:
Same comments as attrs/src/msg.rs, including the ones for QueryMsg.

Reply:
now using `cw_serde`